data/json/items/gun/40.json

```diff
@@ -207,5 +207,42 @@
       [ "stock", 1 ],
       [ "underbarrel", 1 ]
     ]
+  },
+  {
+    "id": "hi_power_40",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Browning Hi Power .40 S&W",
```

Comment:
The Wikipedia article hyphenates Hi-Power in the title, then in the very first sentence, doesn't use a hyphen, then in the third sentence does use a hyphen...
I personally think the hyphen is warranted, but I'll defer to anyone who says otherwise.

Reply:
I'll probably change it back to having a hyphen.